lib/tether/ops_linux.go

```diff
@@ -57,6 +57,7 @@ const (
 	pciDevPath         = "/sys/bus/pci/devices"
 	nfsFileSystemType  = "nfs"
 	ext4FileSystemType = "ext4"
+	bridgeTableNumber  = 201
```

Comment:
```
$ more /etc/iproute2/rt_tables 
#
# reserved values
#
255	local
254	main
253	default
0	unspec
#
# local
#
#1	inr.ruhep
```
As long as we don't use the reserved values, and no one else inserts another table with the same id (low chance of that happening since we own the VCH), we should be fine.